CvGameCoreDLL_Expansion2/CvGame.cpp

```diff
@@ -10772,14 +10772,42 @@ int CvGame::GetCultureMedian() const
 	return m_iCultureMedian;
 }
 
-
 void CvGame::initSpyThreshold()
 {
-	if (!MOD_BALANCE_CORE_SPIES)
+	// Early return if spy balance module is not active
+	if (!MOD_COMMUNITY_PATCH || !MOD_BALANCE_CORE_SPIES)
```

Comment:
MOD_COMMUNITY_PATCH is always true. There'll be mass removal of these kinds of defines later.

Reply:
Just added ASSERT for it instead as always true, can be all removed in future.